apps/next-evans/lib/db/friends.ts

```diff
@@ -1,37 +1,60 @@
 import invariant from 'tiny-invariant';
+import type { Lang } from '@friends-library/types';
 import type { CustomCode } from './custom-code';
-import type { DocumentWithFriendMeta, FriendProps } from '../types';
+import type { DocumentWithMeta, FriendType } from '../types';
 import { LANG } from '../env';
+import { documentRegion, getPrimaryResidence } from '../residences';
 import { prisma } from './prisma';
 import getAllCustomCode from './custom-code';
 
-let friendsPromise: Promise<Record<string, FriendProps>> | null = null;
+let friendsPromise_en: Promise<Record<string, FriendType>> | null = null;
+let friendsPromise_es: Promise<Record<string, FriendType>> | null = null;
```

Comment:
what if we changed this to:

```ts
const friends: {
  en: Promise<Record<string, FriendType>> | null;
  es: Promise<Record<string, FriendType>> | null;
} = { en: null, es: null };
```

I feel like it would clean up the function below.